packages/embed-grid/package.json

```diff
@@ -26,12 +26,12 @@
     "@deephaven/react-hooks": "^0.11.0",
     "@deephaven/utils": "^0.11.0",
     "@fortawesome/fontawesome-svg-core": "1.2.36",
-    "@fortawesome/react-fontawesome": "^0.1.15",
+    "@fortawesome/react-fontawesome": "^0.1.12",
```

Comment:
Downgrade? I think this is the same thing Vlad commented on, just in a different package.json

Reply:
Accident